pkgs/tools/security/clamav/default.nix

```diff
@@ -14,7 +14,7 @@ stdenv.mkDerivation rec {
   };
 
   # Flaky test, remove this when https://github.com/Cisco-Talos/clamav/issues/343 is fixed
-  patches = [ ./remove-freshclam-test.patch ];
+  patches = [ ./remove-freshclam-test.patch ./sample-cofiguration-file-install-location.patch ];
```

Comment:
```suggestion
  patches = [
    # Flaky test, remove this when https://github.com/Cisco-Talos/clamav/issues/343 is fixed
    ./remove-freshclam-test.patch
    ./sample-cofiguration-file-install-location.patch
  ];
```

Reply:
Ah yes, this makes it much more clearer, I didn't consider context